### PR TITLE
OCLOMRS-224:Change and fix the pages' metadata (title) to be relevant

### PR DESCRIPTION
**Pull request by muhozi**
# JIRA TICKET NAME:
[Change and fix the pages' metadata (title) to be relevant](https://issues.openmrs.org/browse/OCLOMRS-224)

# Summary:
For now the title across the page is the same, and the application is using default reactJS favicon. This needs to be fixed so that the title on a page would be relevant to its contents, and also the favicon should be replaced by that of OCL.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
     "react-avatar": "^2.5.1",
     "react-bootstrap": "^0.32.1",
     "react-dom": "^16.3.2",
+    "react-helmet": "^5.2.0",
```
Comment (alexmochu) [inline package.json]: @muhozi What is this library doing?

Comment (muhozi) [reply package.json]: @alexmochu It helps to manage header contents including `title`, `description` and `meta tags` of the page.

**Comment by Duncanian** [comment]
LGTM, but there are conflicts you should resolve.

**Comment by Duncanian** [comment]
LGTM, but there are conflicts you should resolve.

File: src/components/login/container/index.jsx [hunk 2]
```diff
@@ -40,12 +30,24 @@ export class Login extends Component {
     }
   }
 
+  handleInput = (event) => {
+    const { name, value } = event.target;
+    this.setState({ [name]: value });
+  };
```
Comment (dkayiwa) [inline src/components/login/container/index.jsx]: Do all these changes have anything to do with the ticket?

File: src/components/login/container/index.jsx [hunk 3]
```diff
@@ -57,58 +59,47 @@ export class Login extends Component {
...
-                  <input
-                    type="text"
-                    className="form-control"
-                    placeholder="OCL Username"
-                    name="username"
-                    id="username"
-                    required
-                    value={username.trim()}
-                    onChange={this.handleInput}
-                  />
```
Comment (dkayiwa) [inline src/components/login/container/index.jsx]: Do all these changes have anything to do with the ticket?

File: src/components/dashboard/container/OwnerDictionary.jsx [hunk 4]
```diff
@@ -58,14 +62,17 @@ export class OwnerDictionary extends Component {
...
-          organizations={organizations}
-        />
+        <Title title="Login" />
+        <UserDashboard dictionaries={dictionaries} organizations={organizations} />
         <p> Your dictionaries </p>
         <div className="dashboard-head">
-          <div className="btn add-dictionaries" onClick={this.handleShow}>
-              Add Dictionary
+          <div
+            className="btn add-dictionaries"
```
Comment (dkayiwa) [inline src/components/dashboard/container/OwnerDictionary.jsx]: Do all these changes have anything to do with the ticket?

**Comment by dkayiwa** [comment]
There are a number of changes which do not seem to have anything to do with the ticket?

File: src/components/login/container/index.jsx (hunk 2 above)
Comment (muhozi) [reply src/components/login/container/index.jsx]: Thank you @dkayiwa for this. Let me revert all these unrelated changes.

File: src/components/login/container/index.jsx (hunk 3 above)
Comment (muhozi) [reply src/components/login/container/index.jsx]: The same here. Let me revert these unrelated changes.

File: src/components/dashboard/container/OwnerDictionary.jsx (hunk 4 above)
Comment (muhozi) [reply src/components/dashboard/container/OwnerDictionary.jsx]: Same here. Let me revert these unrelated changes.

**Pull request merged**